cmd/zed/dev/vector/run/command.go

```diff
@@ -0,0 +1,86 @@
+package run
+
+import (
+	"errors"
+	"flag"
+
+	"github.com/brimdata/zed"
+	"github.com/brimdata/zed/cli/outputflags"
+	"github.com/brimdata/zed/cmd/zed/dev/vector"
+	"github.com/brimdata/zed/cmd/zed/root"
+	"github.com/brimdata/zed/compiler"
+	"github.com/brimdata/zed/pkg/charm"
+	"github.com/brimdata/zed/pkg/storage"
+	"github.com/brimdata/zed/runtime"
+	"github.com/brimdata/zed/runtime/vcache"
+	"github.com/brimdata/zed/zbuf"
+	"github.com/brimdata/zed/zio"
+	"github.com/segmentio/ksuid"
+)
+
+var run = &charm.Spec{
+	Name:  "run",
+	Usage: "run [flags] query path",
+	Short: "run a Zed query on a VNG file",
```

Comment:
Would it make sense to rename this later to "zed dev vector query" for symmetry with "zed query"?

Reply:
Yes!  I'll change it now...